lib/tasks/whitehall_importer.rb

```diff
@@ -188,3 +220,13 @@ def tags(associations)
 
   class AbortImportError < RuntimeError; end
 end
+
+class RawFile
```

Comment:
I'm not sure RawFile is the right name for this object, perhaps AssetManagerFile ?

What I'd suggest is that we actually give this an input of the Asset Manager URL and then use this class the responsibility of converting that to a file object. We could also use that URL for the filename.

Reply:
I've now done away with this extra class, by configuring ImageBlobService to be more easily callable and saving having to stub `revision` and `original_filename` in hacky ways.